examples/profilesandbox/.env.example

```
@@ -0,0 +1,3 @@
+SANDBOX_CLIENT_SDK_ID=
```

```suggestion
YOTI_SANDBOX_CLIENT_SDK_ID=
```
Should we always prefix with `YOTI_`? Doesn't matter too much as it's the example